### ISSUE TITLE
Autoplay with oembed

### ISSUE THREAD

**Issue by xf-** [voc/voctoweb#553]
I want to embed a video privacy-friendly in a website without loading until somebody presses play. At that moment the iframe source will be set and autoplay will start the video.

**Comment by saerdnaer** [comment]
Modern browsers no longer allow autoplay with sound – there is nothing we can do about that. Sorry.

**Comment by xf-** [comment]
@saerdnaer that's not true or not in all cases. I have a working website with up2date firefox and chromium working. Tested in private/incognito windows. Maybe something is different after it isn't started onload.

I use the attribute autoplay true with no source set. onclick the source is set.
https://developer.mozilla.org/en-US/docs/Web/Media/Autoplay_guide#example_allowing_autoplay_from_specific_sources

You are in matrix or IRC? I would like to show you an example.

**Comment by saerdnaer** [comment]
Sure you find me andi-m@matrix.org – but feel free to post code examples here in this thread or open an PR with the necessary changes.
